pkg/exporter/sumologicexporter/sender.go

```diff
@@ -324,19 +324,16 @@ func (s *sender) handleReceiverResponse(resp *http.Response) error {
 
 func (s *sender) createRequest(ctx context.Context, pipeline PipelineType, data io.Reader) (*http.Request, error) {
 	var url string
-	if s.config.HTTPClientSettings.Endpoint == "" {
```

Comment:
It's the reverse. Even if the endpoint is not empty, we still use the signal-specific urls. This actually doesn't change anything, because these values were the same; that is, if endpoint was set, the signal-specific urls were equal to it.